src/test/resources/projects/standard-project/pom.xml

```diff
@@ -14,30 +14,76 @@
   <packaging>war</packaging>
   <name>App Engine Standard Project</name>
 
-  <build>
-    <plugins>
-      <plugin>
-        <groupId>com.google.cloud.tools</groupId>
-        <artifactId>appengine-maven-plugin</artifactId>
-        <version>${appengine-maven-plugin.version}</version>
-        <configuration>
-          <version>1</version>
-          <project>travis-app-maven-plugin</project>
-          <version>GCLOUD_VERSION</version>
-          <jvmFlags>
-            <!-- required for kokoro gcp_ubuntu b/37217450 -->
-            <jvmFlag>-Djava.security.egd=file:/dev/urandom</jvmFlag>
-          </jvmFlags>
-          <environment>
-            <TEST_VAR>testVariableValue</TEST_VAR>
-          </environment>
-          <project>travis-app-maven-plugin</project>
-        </configuration>
-      </plugin>
-    </plugins>
-  </build>
 
   <profiles>
+    <profile>
+      <id>java8_and_below</id>
+      <activation>
+        <jdk>[,8]</jdk>
+      </activation>
+      <build>
+        <plugins>
+          <plugin>
+            <groupId>com.google.cloud.tools</groupId>
+            <artifactId>appengine-maven-plugin</artifactId>
+            <version>${appengine-maven-plugin.version}</version>
+            <configuration>
+              <version>1</version>
+              <project>travis-app-maven-plugin</project>
```

Comment:
Is it possible to change the project name here? I'm guessing it is specific to the CI.

Reply:
It would be good to change at some point, since it likely is a throwback to when we were on Travis CI for many projects. But I did not introduce it in this PR, so I don't want to mess with it.